frontend/components/category-headings.tsx

```diff
@@ -1,24 +1,26 @@
 import React from 'react'
-import PropTypes from 'prop-types'
 import styled from 'styled-components'
 import { Link } from 'gatsby'
 import { H3 } from './common'
 
-export function CategoryHeading({ category: { id, name } }) {
+export interface Category {
```

Comment:
More of a strategy question, but is the thinking with the switch to TS for any type (interfaces, classes, aliases, etc.) that's only used in one file (presumably like `Category` here) to define that within the file, and then for a type used in multiple places that we'll define that in something like `common.tsx`?